tf_glue_barebones.py

```diff
@@ -0,0 +1,87 @@
+import logging
+from dataclasses import dataclass, field
+
+from data_processors import DataProcessorForSequenceClassification
+from tf_training_args import TFTrainingArguments
+from trainer_tf_barebones import TFTrainer
+from transformers import AutoConfig, AutoTokenizer
+from transformers import TFAutoModelForSequenceClassification
+
+
+logging.basicConfig(level=logging.INFO)
+
+
+
+@dataclass
+class ModelArguments:
+    model_name_or_path: str = field(
+        metadata={"help": "Path to pretrained model or model identifier from huggingface.co/models"}
+    )
+
+
+@dataclass
+class DataTrainingArguments:
+    task_name: str = field(metadata={"help": "The name of the task to train on"})
+    data_dir: str = field(
+        metadata={"help": "The input data dir. Should contain the .tsv files (or other data files) for the task."}
+    )
+    max_seq_length: int = field(
+        default=128,
+        metadata={
+            "help": "The maximum total input sequence length after tokenization. Sequences longer "
+            "than this will be truncated, sequences shorter will be padded."
+        },
+    )
```

Comment:
These fields cannot be part of `TrainingArguments`? Or it doesn't make sense and we should keep them outside?